content/en/docs/user-documentation/text-editor/building-buttons/_index.md

```diff
@@ -31,18 +34,18 @@ This screen gives you basic options to style your link or button. On the top lef
 
 The top right “Size” dropdown four options for your button size. If you chose “Link” style option, the Size option will not affect your link.
 
-![blog-destiption__text-editor-button_style|166x500,50%](../../../../../assets/img/450d40d931e05bc00fe0d3b6d15d951b0062793d.png)  
-![blog-destiption__text-editor-button_size|282x200,50%](../../../../../assets/img/a27170cb92cff52b1a9bd2590993d413b62350c7.png)
+![blog-destiption__text-editor-button_style|166x500,50%](buttons--style.png)
+![blog-destiption__text-editor-button_size|282x200,50%](buttons--size.png)
 
 Add the text for your link/button in the bottom left. Enter your link in the URL field on the bottom right.
 
 * For links on your website, don’t use the full URL. Highlight everything beginning with the `/` after your `.com`, `.org`, etc.
-  * For example, for openy.org/about, you would choose `/about`. This is called the relative path, and it will help your analytics tracking.
+  * For example, for exampleymca.org/about, you would choose `/about`. This is called the relative path, and it will help your analytics tracking.
```

Comment:
```suggestion
  * For example, for example.org/about, you would choose `/about`. This is called the relative path, and it will help your analytics tracking.
```